### PR TITLE
feature/BG-27

### PR DESCRIPTION
**Pull request by MaciekPon**
feat(authors):

- change styles for desktop
- add styles for mobile

### CODE REVIEW

File: src/authors/authors.html [hunk 1]
```diff
@@ -1,35 +1,41 @@
...
-        
-        </main>
- </body>
-
-</html>
\ No newline at end of file
+  <body>
+    <main class="main">
+      <nav class="pop">
+        <h1 class="title">Authors</h1>
```
Comment (ruljin) [inline src/authors/authors.html:13]: It is not a mistake cause `<nav>` can have other content with links that only `<ul>` or `<ol>` like for example text with `<hX>` and `<p`> with links but for consistency what would you say, to move the header out of `<nav`> if it is not a textual content but a list?

File: src/authors/authors.html [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
+            <a class="link--github" href="https://github.com/alicjaguzik" target="_blank">Development Manager: Alicja Guzik</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/MaciekPon" target="_blank">Maciej Ponicki</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/WelcomeInFront" target="_blank">Justyna Rozbicka</a>
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/DawidZlobecki" target="_blank">Dawid Złobecki</a>
```
Comment (ruljin) [inline src/authors/authors.html]: Żłobecki not Złobecki

**Review by ruljin — CHANGES_REQUESTED** [review]

File: src/authors/authors.html (hunk 1 above)
Comment (ruljin) [reply src/authors/authors.html:13]: The space between `<h1>` and first `<li>` is other than the one on Figma

File: src/authors/authors.html [hunk 3]
```diff
@@ -1,35 +1,41 @@
...
+          </li>
+          <li class="menu__list-item">
+            <a class="link--github" href="https://github.com/ruljin" target="_blank">Mentor: Filip Kuca</a>
+          </li>
+        </ul>
+        <a class="return" href="index.html">Return</a>
```
Comment (ruljin) [inline src/authors/authors.html]: I think, `Return` should be at the bottom of the right corner like on other pages. Right now it is on the bottom of the container, which is not always height enough to place it on the bottom.

File: src/authors/authors.html (hunk 1 above)
Comment (MaciekPon) [reply src/authors/authors.html:13]: so i will create `<section>` with `<h1>` outside the `<nav>` and I will see what it looks like

**Review by ruljin — APPROVED** [review]

**Pull request merged**